src/imex/exceljs.ts

```diff
@@ -10,5 +8,6 @@ export const getCellValueTypes = async () => {
 
 export const createWorkbook = async () => {
   const excelJS = await getExcelJs()
-  return new excelJS.Workbook() as Excel.Workbook
+
+  return new excelJS.Workbook()
```

Comment:
I did this because the typing was broken with your approach, does it work now ?

Reply:
The language server seems to be happy :man_shrugging: